chore(styles): apply header list styles to buttons (#4785)
swisspost/design-system: .changeset/nervous-glasses-pay.md

---
'@swisspost/design-system-styles': patch
---

> Updated header list styles to also be applied to buttons.

Reviewer: ```suggestion
Updated header button styles to match the header anchor styles.
```
